### PR TITLE
test top book borrow

### PR DESCRIPTION
**Pull request by at-phutran2**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-1
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-tamnguyen4** [comment]
Thiếu test về pagination nghe a

File: tests/Feature/TopBookApiTest.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    /**
+     * Test result pagination.
+     *
+     * @return void
+     */
+    public function testWithPaginationTopBorrowBooks()
+    {
+        $this->makeListOfBook(21);
+        $response = $this->json('GET', '/api/books/top-borrow' . '?page=2');
+//        dd($response);
```
Comment (at-tamnguyen4) [inline tests/Feature/TopBookApiTest.php]: Xóa comment code không cần thiết đi a

File: tests/Feature/TopBookApiTest.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        $response->assertJson([
+            'data' => [
+            'current_page' => 2,
```
Comment (at-tamnguyen4) [inline tests/Feature/TopBookApiTest.php]: đoạn này thêm space đi a, cho rõ ràng

**Comment by at-tamnguyen4** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**